url-shortener-server/url_shortener_server/base_routers.py

```diff
@@ -0,0 +1,96 @@
+from abc import ABC, abstractmethod
+from importlib import import_module
+
+from flask.views import View
+
+
+class ImproperlyConfigured(Exception):
+    pass
+
+
+class BaseRouter(ABC):
+    """https://github.com/thisissoon/Flask-Via/blob/master/flask_via/routers/__init__.py#L13"""
+
+    @abstractmethod
+    def __init__(self):
+        raise NotImplementedError("__init__ must be overridden")
```

Comment:
Since it's abstractmethod' we can:
```suggestion
        """ Add ... logic to this method """
```